geospace-http/src/main/resources/application.conf

```diff
@@ -37,6 +37,15 @@ com.socrata {
       domains = ["regions.test.socrata.com"]
       bounding-shape-precision = 0.99
     }
+
+    partitioning {
+      # Controls the size of a single partition of region shapes.  Shapes are loaded one partition at a time,
+      # as many point datasets only use a subset of large national shapes like zip codes.
+      # These are in WGS84 coords, so sizeX corresponds to longitude and sizeY corresponds to latitude
+      # At the 45th parallel, 5 deg longitude = 78km and 5 deg latitude = 111 km.
+      sizeX = 5.0
```

Comment:
Are you sure on those degree to km figures?  At least for latitude my back of the envelope says 5 degrees * 1 nautical mile/minute * 60 minutes/degree * ~1.15 nautical miles/mile * ~1.6km/mile =~ 550km.